packages/atlogger/src/atlogger.c

```diff
@@ -62,7 +124,11 @@ static void atlogger_get_prefix(enum atlogger_logging_level logging_level, char
   }
   }
 
-#ifndef ATLOGGER_DISABLE_TIMESTAMPS
+#if !defined(ATLOGGER_DISABLE_TIMESTAMPS)
+#if (_POSIX_VERSION < 200112L)
+  time_t ts = time(NULL);
+  off += snprintf(prefix + off, PREFIX_BUFFER_LEN - off, " %" PRIu64, (uint64_t)ts);
+#else
```

Comment:
If timestamps are enabled on a build machine that doesn't implement POSIX.1-2001 we use the more portable time function which is only accurate to the second.